### PR TITLE
integration test for  BQSR Plotting 

### PR DESCRIPTION
**Pull request by akiezun**
https://github.com/broadinstitute/hellbender/issues/322

@droazen please review

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java [hunk 1]
```diff
@@ -74,6 +79,34 @@ public void testBQSR(BQSRTest params) throws IOException {
         spec.executeTest("testBQSR-" + params.args, this);
     }
 
+    @Test(description = "This is to test https://github.com/broadinstitute/hellbender/issues/322")
+    public void testPipeline() throws IOException {
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java]: This test needs a better name like "testPlottingWorkflow"

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java [hunk 2]
```diff
@@ -74,6 +79,34 @@ public void testBQSR(BQSRTest params) throws IOException {
...
+        new ApplyBQSR().instanceMain(Utils.escapeExpressions(argApply));
+
+        final File actualTablePost = createTempFile("gatk4.post.cols", ".table");
+        final String argsPost = "-R " + hg18Reference + " --knownSites " + dbSNPb37 + " -I " + actualHiSeqBam_recalibrated.getAbsolutePath() + " -RECAL_TABLE_FILE " + actualTablePost.getAbsolutePath() + " --sort_by_all_columns true";
+        new BaseRecalibrator().instanceMain(Utils.escapeExpressions(argsPost));
+
+        final File expectedHiSeqBam_recalibrated = new File(resourceDir + "expected.NA12878.chr17_69k_70k.dictFix.recalibrated.bam");
+
+        //this fails, disable for now: https://github.com/broadinstitute/hellbender/issues/419
+        //IntegrationTestSpec.compareBamFiles(actualHiSeqBam_recalibrated, expectedHiSeqBam_recalibrated);
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java]: Add a note in https://github.com/broadinstitute/hellbender/issues/419 to uncomment these lines once that issue is resolved.

**Comment by droazen** [comment]
Two small issues, then good to merge.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java (hunk 2 above)
Comment (akiezun) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java]: done

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java (hunk 1 above)
Comment (akiezun) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java]: done

**Pull request merged**